xformers/factory/model_factory.py

```diff
@@ -30,6 +30,43 @@ class xFormerConfig:
     the encoder and decoder positional encoding, as proposed for instance by
     `Using the Output Embedding to Improve Language Models`, Press et al.
 
+    A full config example is for instance as follows:
```

Comment:
opportunist improvement, now looks like this in the docs
![Screenshot from 2022-01-07 10-26-13](https://user-images.githubusercontent.com/3072861/148589682-dff21ea2-656a-4d1b-91e5-e0441d162a00.png)